File: 1 - INICIANTE/C++17/1020.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std; 


int main() {
int n,a,b,c;
cin >> n;

a = (n/365);

b = (n%365)/30;

c = (n%365)%30;

cout << a << " ano(s)" << "\n" << b << " mes(es)" << "\n" << c << " dia(s)" << "\n";
return 0;

}

File: 1 - INICIANTE/python/1020.py
entrada_dias = int(input())
anos = int(entrada_dias/365)
meses = int(entrada_dias%365/30)
dias = int(entrada_dias%365%30)

print('{} ano(s)\n{} mes(es)\n{} dia(s)'.format(anos, meses, dias))